packages/grafana-llm-app/pkg/plugin/provider.go

```diff
@@ -6,6 +6,8 @@ func createProvider(settings *Settings) (LLMProvider, error) {
 	switch settings.OpenAI.Provider {
 	case openAIProviderOpenAI:
```

Comment:
nit: you can do `case openAIProviderOpenAI, openAIProviderCustom:` here if you like!